.gitignore

```diff
@@ -31,3 +31,12 @@ build/
 
 ### VS Code ###
 .vscode/
+
+
+# Ignore application properties
+application.properties
```

Comment:
application.properties should no longer be in the .gitignore because it no longer contains actual data instead we use a .env file now